src/services/assetService.test.ts

```diff
@@ -19,6 +19,18 @@ describe("Asset Service", () => {
             expect(asset.format).toEqual("jpg");
         });
 
+        it("creates an asset from an encoded file", () => {
```

Comment:
This is a good test - but it is not testing the change you made.  We'll need to test that calls into `getAssets()` so we can assert that the path has been encoded when retrieved by the app.

Reply:
I'm on it. I may be coming to you with questions